src/sparseml/onnx/utils/graph_editor.py

```diff
@@ -450,3 +451,58 @@ def prune_model_one_shot_iter(
         pruned_weight_val = prune_unstructured(weight.val, sparsity)
         update_model_param(model, weight.name, pruned_weight_val)
         yield (index + 1) / len(nodes)
+
+
+def find_orphaned_nodes(model: ModelProto, node: NodeProto) -> List[NodeProto]:
```

Comment:
would be nice to make this a method of `ONNXGraph`

Reply:
Sounds good